.buildkite/release-pipelines/start-code-freeze.yml

```diff
@@ -22,9 +22,6 @@ steps:
 
       echo '--- :snowflake: Start Code Freeze'
       bundle exec fastlane start_code_freeze skip_confirm:true
-
-      echo '--- :cocoapods: Check for Outdated Pods'
-      .buildkite/commands/log-outdated-pods.sh
```

Comment:
This well need to be followed up with an update on the Releases v2 app to remove the CocoaPods mentions there, too.